manifests/init.pp

```diff
@@ -280,6 +285,14 @@
       hostname => $capsule_fqdn,
       require  => Package['foreman-proxy'],
       before   => Service['foreman-proxy'],
+    } ~>
+
+    # Deploy client certificates for foreman proxy to talk to master
+    class { 'certs::foreman':
+      configure_foreman => false,
+      owner             => 'foreman-proxy',
+      client_cert       => $foreman_client_cert,
+      client_key        => $foreman_client_key,
     }
```

Comment:
Actually this probably blows up in the case when installing the dependent capsule, will have to think of another way to do it, not sure how exactly :-\

I want to use RPM's already in the certs-tar on existing capsules.